_releases/2022-01-06-1.3.0-released.md

```diff
@@ -22,6 +22,21 @@ In this release we included [XXX PRs since the 1.2.2 release](https://github.com
 
 ## Multi-Assign
 
+Multiple assignments got improved in various ways. First, it's possible to take a "splat of elements" in a multi-assignment:
+
+<div class="code_section">{% highlight crystal %}
+n_first, *n_s, n_last = [1, 2, 3, 4, 5]       # n_s is [2, 3, 4]
+*t_s, t_last = {"This", 15, 4, "tuple", true} # t_s is {"This", 15, 4, "tuple"}
+{% endhighlight %}</div>
+
+Second, it's possible to detect unbalanced multi-assignments:
+
+<div class="code_section">{% highlight crystal %}
+not, that, many = [1, 2]    # Multiple assignment count mismatch (IndexError)
```

Comment:
This is a bad example because the flag doesn't have much effect at runtime (for typical `Indexable` implementations, that is).
Both enabled and disabled raises `IndexError`. Only difference is the message (`Multiple assignment count mismatch` vs. `Index out of bounds`) and location (user code vs. `Indexable#[]`).
I don't think this part of the change is even *worth* mentioning because if that.

Instead, we should use an example of a compile time error. Borrowed and updated from the PR:
```suggestion
a, b, c = {1, 2} # Error: index out of bounds for Tuple(Int32, Int32) (2 not in -2..1)
a, b = {1, 2, 3} # Error: cannot assign Tuple(Int32, Int32, Int32) to 2 targets
```

Reply:
Btw. that first error message looks odd. I don't think that's intended 🤔 
It should be `cannot assign Tuple(Int32, Int32) to 3 targets`.

Reply:
https://github.com/crystal-lang/crystal/issues/11695